### DISCUSSION TITLE
Change port with docker install

### DISCUSSION THREAD

**Discussion by pvh0** [fuatakgun/eufy_security#307] [Q&A] [answered]
I've mounted the docker image but unfortunately I already have zwave2mqtt running also on port 3000 so it won't start. Next thing I did is change the port mapping in docker to 3001:3000 and then it starts, logs in successfully to Eufy (states I have 0 devices but will fix that later). 
Unfortunately my Home Assistant then does not pick it up as an integration, probably because it is at port 3001. So when I add integration I can't find "Eufy Security". Any idea how I could fix this?

**Comment by gregf888** [comment]
Are you adding "Eufy Security" using HACS first then add it again as a HA Integration.  You wont see it as a HA integration until you add it with HACS.

**Comment by pvh0** [comment, marked as answer]
Update: Got it now working with version 0.9.2 of the Eufy Security Add-on. Instructions to get this done with Docker on Synology:
- docker run -it -e USERNAME=email@address.com -e PASSWORD=password_goes_here -e COUNTRY=country_code -p 3002:3000 bropat/eufy-security-ws:X.Y.Z

To find out correct values for X.Y.Z, please check here https://github.com/fuatakgun/eufy_security_addon/blob/main/config.json#L3
In this example I chose free port 3002. Check in the docker logs if the container runs ok. If not check Eufy username and password.

Then copy the "custom_components/eufy_security" from this github to your home assistant custom components. Restart Home Assistant and click add integration, choose port 3002 instead of 3000. Then it should work (to be sure check the HA logs).
